python/python/bystro/search/utils/tests/test_annotation.py

```diff
@@ -0,0 +1,240 @@
+    assert "More than 1 config path found, choosing first" in captured.out
+    assert result == mocked_paths[0]
+
+
+def test_statistics_output_extensions_defaults():
+    extensions = StatisticsOutputExtensions()
+    assert extensions.json == "statistics.json"
+    assert extensions.tsv == "statistics.tsv"
+    assert extensions.qc == "statistics.qc.tsv"
+
+
+def test_statistics_config_defaults():
+    config = StatisticsConfig()
+    assert config.dbSNPnameField == "dbSNP.name"
+    assert config.siteTypeField == "refSeq.siteType"
+    assert config.exonicAlleleFunctionField == "refSeq.exonicAlleleFunction"
+    assert config.refField == "ref"
+    assert config.homozygotesField == "homozygotes"
+    assert config.heterozygotesField == "heterozygotes"
+    assert config.altField == "alt"
+    assert config.programPath == "bystro-stats"
+    assert isinstance(config.outputExtensions, StatisticsOutputExtensions)
+
+
+def test_statistics_config_from_dict_none():
+    config = StatisticsConfig.from_dict()
+    assert (
+        config.dbSNPnameField == "dbSNP.name"
+    )  # Check one of the default values as a representative
+
+
+def test_statistics_config_from_dict_no_statistics_key():
+    annotation_config = {"random_key": "random_value"}
+    config = StatisticsConfig.from_dict(annotation_config)
+    assert (
+        config.dbSNPnameField == "dbSNP.name"
+    )  # Again, just checking one representative default value
+
+
+def test_statistics_config_from_dict_with_statistics_key():
+    annotation_config = {
+        "statistics": {
+            "dbSNPnameField": "new.dbSNP.name",
+            "siteTypeField": "new.refSeq.siteType",
+            "outputExtensions": {
+                "json": ".new.json",
+                "tab": ".new.tsv",
+                "qc": ".new.qc.tsv",
+            },
+        }
+    }
+    config = StatisticsConfig.from_dict(annotation_config)
+    assert config.dbSNPnameField == "new.dbSNP.name"
+    assert config.siteTypeField == "new.refSeq.siteType"
+    assert config.outputExtensions.json == ".new.json"
+    assert config.outputExtensions.tsv == ".new.tsv"
+    assert config.outputExtensions.qc == ".new.qc.tsv"
+
+
+def test_statistics_init_program_not_found(mocker):
+    mocker.patch("shutil.which", return_value=None)
+    with pytest.raises(ValueError, match=r"Couldn't find statistics program"):
+        Statistics("/dummy/path", None)
+
+
+def test_statistics_get_stats_arguments(mocker):
+    # Mocking which to return a valid program path
+    mocker.patch("shutil.which", return_value="/path/to/bystro-stats")
+
+    statistics = Statistics("/dummy/output_base_path", None)
+
+    # Note: Your actual values might differ based on the defaults
+    # in StatisticsConfig and DelimitersConfig
+    expected_args = (
+        "/path/to/bystro-stats -outJsonPath /dummy/output_base_path.statistics.json "
+        "-outTabPath /dummy/output_base_path.statistics.tsv "
```

Comment:
should these be long flags here?  (ps disregard if it's correct as written)

Reply:
Correct as written: https://pkg.go.dev/flag